pkg/apis/core/validation/shoot.go

```diff
@@ -271,6 +271,18 @@ func ValidateShootSpecUpdate(newSpec, oldSpec *core.ShootSpec, newObjectMeta met
 
 	allErrs = append(allErrs, validateDNSUpdate(newSpec.DNS, oldSpec.DNS, newSpec.SeedName != nil, fldPath.Child("dns"))...)
 	allErrs = append(allErrs, validateKubernetesVersionUpdate(newSpec.Kubernetes.Version, oldSpec.Kubernetes.Version, fldPath.Child("kubernetes", "version"))...)
+
+	if len(newSpec.Kubernetes.Version) != 0 && len(oldSpec.Kubernetes.Version) != 0 &&
+		versionutils.ConstraintK8sGreaterEqual125.Check(semver.MustParse(newSpec.Kubernetes.Version)) &&
+		versionutils.ConstraintK8sLess125.Check(semver.MustParse(oldSpec.Kubernetes.Version)) {
+
+		pspDisabledInNewSpec := isPSPDisabled(newSpec.Kubernetes.KubeAPIServer)
+		pspDisabledInOldSpec := isPSPDisabled(oldSpec.Kubernetes.KubeAPIServer)
+		if !pspDisabledInNewSpec || !pspDisabledInOldSpec {
+			allErrs = append(allErrs, field.Forbidden(fldPath.Child("kubernetes").Child("version"), fmt.Sprintf("admission plugin: %q should be disabled for kubernetes version >=1.25, Refer [Migrating to PodSecurity](https://github.com/gardener/gardener/blob/master/docs/usage/pod-security.md#migrating-to-podsecurity)", "PodSecurityPolicy")))
```

Comment:
```suggestion
			allErrs = append(allErrs, field.Forbidden(fldPath.Child("kubernetes").Child("version"), `admission plugin "PodSecurityPolicy" should be disabled for Kubernetes versions >=1.25, please check https://github.com/gardener/gardener/blob/master/docs/usage/pod-security.md#migrating-to-podsecurity`))
```

There is no markdown in such error-messages 😉